src/js/controllers/walletHome.js

```diff
@@ -4,7 +4,7 @@
 // TODO rateService
 angular.module('copayApp.controllers').controller('walletHomeController', function($scope, $rootScope, $timeout, $filter, $modal, notification, txStatus, isCordova, profileService) {
 
-
+  $rootScope.$emit('updatePendingTxs');
```

Comment:
no sure if we want to emit this event everytime the homepage loads, but we can optimise this later.